### PR TITLE
Add device class for low battery

### PR DESCRIPTION
**Pull request by schmittx**
- Adds `battery` device class
- [Related PR](https://github.com/home-assistant/home-assistant/pull/10829)

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -32,6 +32,10 @@
       "on": "[%key:state::default::on%]"
     },
     "binary_sensor": {
+      "battery": {
```
Comment (arsaboo) [inline src/translations/en.json]: Put this below the `default` binary_sensor`.

Comment (schmittx) [reply src/translations/en.json]: Good catch, done.

**Comment by schmittx** [comment]
All of the related PRs have been merged, can we get this merged also and included with 0.59 this weekend?

**Review by emlove — APPROVED** [review]

File: src/translations/en.json [hunk 2]
```diff
@@ -36,6 +36,10 @@
         "off": "[%key:state::default::off%]",
         "on": "[%key:state::default::on%]"
       },
+      "battery": {
+        "off": "Normal",
```
Comment (armills) [inline src/translations/en.json]: I'd suggest using "OK" here instead of "Normal". If you do, use the key referencing syntax `[%key:state::plant::ok%]`.

Comment (schmittx) [reply src/translations/en.json]: Good suggestion, consistency is best!

File: src/util/hass-attributes-util.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
   binary_sensor: [
     'connectivity', 'light', 'moisture', 'motion', 'occupancy', 'opening',
     'sound', 'vibration', 'gas', 'power', 'safety', 'smoke', 'cold', 'heat',
-    'moving', 'plug', 'presence'],
+    'moving', 'plug', 'presence', 'battery'],
```
Comment (andrey-git) [inline src/util/hass-attributes-util.html]: Please sort

Comment (schmittx) [reply src/util/hass-attributes-util.html]: Done, I also went back and sorted src/translations/en.json

**Review by andrey-git — APPROVED** [review]

**Pull request merged**